pages/common/join.md

```diff
@@ -0,0 +1,7 @@
+# join
+
+> Join lines of two sorted files on a common field.
```

Comment:
`sorted files`? What's meant by this?

Reply:
The files that we try to join should be sorted.

Reply:
@sbrl I don't think this is available in the windows, but all other variants (*nix) have it.
So that was the rationale to put it there